shared/util_linux.go

```diff
@@ -486,8 +486,8 @@ func ExecReaderToChannel(r io.Reader, bufferSize int, exited <-chan struct{}, fd
 				// Read end has likely been closed so again,
 				// avoid an endless loop.
 			} else if (revents & unix.POLLNVAL) > 0 {
-				logger.Warnf("Detected poll(POLLNVAL) event.")
-				// Well, someone closed the fd havent they? So
+				logger.Debugf("Detected poll(POLLNVAL) event.")
```

Comment:
Can you remove the trailing period while you're at it to line it up with our normal logging style?

Reply:
Done